components/landlord-setup/listing-confirmation.tsx

```
@@ -0,0 +1,427 @@
+import { Card } from "@/components/ui/card";
+import { Button } from "@/components/ui/button";
+import { Separator } from "@/components/ui/separator";
+import { ScrollArea } from "@/components/ui/scroll-area";
+import { Input } from "@/components/ui/input";
+import { UploadIcon, Loader2 } from "lucide-react";
+
+interface ListingConfirmationProps {
+	onNext: (data: { photos: string[] }) => void;
+	onBack: () => void;
+	formData: {
+		property: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)